vpp/Makefile.am

```diff
@@ -1,3 +1,5 @@
+INCLUDES = -I$(top_srcdir)/interface
+
```

Comment:
It seems PR #566 was closed. Will the author fix the automake issue in his new pull request?